extensions/wikia/Captcha/CaptchaHooks.class.php

```diff
@@ -58,4 +58,12 @@ public static function APIGetAllowedParams( &$module, &$params ) {
 	public static function APIGetParamDescription( &$module, &$desc ) {
 		return Factory\Module::getInstance()->APIGetParamDescription( $module, $desc );
 	}
+
+	// Temporarily load captcha library on every page until we have on demand loading set up.
+	// Remove with https://wikia-inc.atlassian.net/browse/SOC-288
+	public static function onSkinAfterBottomScripts( $skin, &$text ) {
```

Comment:
Fair point @inez. No particular reason other than we had them ticketed out separately since they fell into somewhat discrete tasks. Loading them on demand wasn't too hard to implement though so I went ahead and added it to this PR.